frontend/app/controllers/home.js

```diff
@@ -1,4 +1,48 @@
-import Controller from '@ember/controller';
-
+import Controller from "@ember/controller";
```

Comment:
you can 
```
import Ember from 'ember';

then you can inject 
//examples 
const {
  computed,
  inject: {
     service
}
  computed: {
    equal
  }
} = Ember;
```

Reply:
Fixed in 0cd4debab433260ab6a8267aa0b72b31db581edb